XLForm/XL/Cell/XLFormSelectorCell.m

```diff
@@ -274,6 +283,8 @@ -(void)formDescriptorCellDidSelectedWithFormController:(XLFormViewController *)c
                                                                   }]];
             }
             [formViewController presentViewController:alertController animated:YES completion:nil];
+            // Fix for "Snapshotting a view that has not been rendered results in an empty snapshot." bug
+            [[alertController view] layoutIfNeeded];
```

Comment:
Yes that is fair enough. I have included a new commit (fbc08a7) in this PR that removes this particular fix. Thanks for the review.